uber/config.py

```diff
@@ -315,6 +315,12 @@ def _is_intstr(s):
         return s[1:].isdigit()
     return s.isdigit()
 
+if not c.KIOSK_CC_ENABLED:
+    del _config['enums']['door_payment_method']['stripe']
```

Comment:
We'll still run into the issue that doing it this way will cause ``c.STRIPE`` and ``c.GROUP`` to not be defined unless they also happen to be defined in other enum subsections.  That's okay if they are defined elsewhere or if we're not referencing them anywhere.